doc/api/addons.markdown

```diff
@@ -1077,14 +1078,14 @@ Test in JavaScript by running:
 const addon = require('./build/Release/addon');
 ```
 
-[online]: https://v8docs.nodesource.com/
```

Comment:
@stevemao well, @thefourtheye gave me this idea in #5002 and I find it good :wink: